addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/RemoteFileIncludeScanRule.java

```diff
@@ -279,4 +276,19 @@ public int getCweId() {
     public int getWascId() {
         return 5;
     }
+
+    private AlertBuilder createAlert(String param, String attack, String evidence) {
+        return newAlert()
+                .setConfidence(Alert.CONFIDENCE_MEDIUM)
+                .setParam(param)
+                .setAttack(attack)
+                .setEvidence(evidence);
+    }
+
+    @Override
+    public List<Alert> getExampleAlerts() {
+        return List.of(
+                createAlert("param", "http://" + "www.google.com/", "<title>Google</title>")
```

Comment:
Use a single string?

Reply:
I had done it that way just to keep things clear, that’s what’s done when the rule constructs things.

Reply:
To be clear I'm suggesting `"http://www.google.com/"`.

Reply:
I know, I was trying to stay consistent with:
- https://github.com/zaproxy/zap-extensions/blob/0c0cc88884246639a7ebb5643b30c9851feaf019/addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/RemoteFileIncludeScanRule.java#L210
- https://github.com/zaproxy/zap-extensions/blob/0c0cc88884246639a7ebb5643b30c9851feaf019/addOns/ascanrules/src/main/java/org/zaproxy/zap/extension/ascanrules/RemoteFileIncludeScanRule.java#L241

Reply:
I see, I think it's fine using a single string anyway.

Reply:
Okay